libtenzir/include/tenzir/multi_series.hpp

```diff
@@ -92,6 +94,45 @@ class multi_series {
     return result;
   }
 
+  auto clear() -> void {
+    return parts_.clear();
+  }
+
+  auto append(series s) -> void {
+    parts_.push_back(std::move(s));
+  }
+
+  // What to do on join conflict in `to_series`
+  enum class to_series_strategy {
+    // Fail the join
+    fail,
+    // Take the first type, null the mismatches
+    take_first_null_rest,
+    // Try to from the largest join, null the mismatches
+    take_largest_null_rest,
+  };
+
+  struct to_series_result {
+    enum class status_t {
```

Comment:
Just call this `status` and use `enum status status` below to get rid of the conflict. Much easier on the eyes and more in line with our other code.